app/models/api/EligibilitySubmissionData.scala

```diff
@@ -29,6 +29,7 @@ case class EligibilitySubmissionData(threshold: Threshold,
                                      customerStatus: CustomerStatus,
                                      partyType: PartyType,
                                      registrationReason: RegistrationReason,
+                                     togcBlock: Option[TogcBlock] = None,
```

Comment:
Minor point, but we tend not to use "block" in the naming of parts of the model - would be better to replace Block with Cole to reference that the block will be used by both.